tests/containers/containers_3rd_party.pm

```diff
@@ -64,8 +64,10 @@ sub run {
     my ($running_version, $sp, $host_distri) = get_os_release;
 
     # Define test images here
-    my @docker_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
-    my @podman_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
+    my $ex_reg        = get_var('REGISTRY', 'docker.io') . '/library';
+    my $os_reg        = 'registry.opensuse.org/opensuse';
+    my @docker_images = ("$ex_reg/alpine", "$os_reg/leap", "$os_reg/tumbleweed", "$ex_reg/debian", "$ex_reg/ubuntu", "$ex_reg/centos", "$ex_reg/fedora");
+    my @podman_images = ("$ex_reg/alpine", "$os_reg/leap", "$os_reg/tumbleweed", "$ex_reg/debian", "$ex_reg/ubuntu", "$ex_reg/centos", "$ex_reg/fedora");
```

Comment:
Unrelated cleanup:

```suggestion
    my @podman_images = @docker_images;
```

Or just change it everywhere to just `@images`...

Reply:
done